spec/integration.spec.js

```diff
@@ -12,6 +12,7 @@ class Todo extends Model {
   static endpoint = 'todos'
 
   @attribute(String) title = ''
+  @attribute(Object) options
```

Comment:
Attribute needed to be defined now in order for the "integration" test to work properly.

Reply:
Is options going to be required for all models now? Like completions and crop batches?

Reply:
I think this is just because we use `options` down in the `can edit an existing model` test.